shared/src/constants/externalAddresses.json

```diff
@@ -20,6 +20,7 @@
       "wsteth": "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
       "ldo": "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
       "aave": "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
+      "uni": "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
```

Comment:
Although it's dummy data since Kovan is deprecated, since we have it for the other vaults should we add "rUNI-THETA-v2": "0x9467726c3cFe3f9112Cd0fA3963223d5b8b3C4B1" as well?

Reply:
same here they are unused